.coafile

```diff
@@ -17,3 +17,7 @@ enabled = False
 files = **/*.rst, README.rst
 ignore = venv/**
 bears = InvalidLinkBear
+
+[rstcheck]
+bears = RSTcheckBear
+files = *.rst,**/*.rst
```

Comment:
yeah. Instead of this. If I do just only **/*.rst. Would mean it would check the directories as well as the sub-directories.